architecture/core_concepts/routes.adoc

```diff
@@ -12,41 +12,83 @@ toc::[]
 
 == Overview
 
-An OpenShift route is a way to expose a
-link:pods_and_services.html#services[service] by giving it an
-externally-reachable hostname like `www.example.com`.
-
-A defined route and the endpoints identified by its service can be consumed by a
-router to provide named connectivity that allows external clients to reach your
-applications. Each route consists of a route name (limited to 63 characters),
-service selector, and (optionally) security configuration.
+An {product-title} link:../../architecture/core_concepts/routes.html#[route] exposes a
+link:../../architecture/core_concepts/pods_and_services.html#services[service] at a
+host name, like _www.example.com_, so that external clients can reach it by
+name.
+
+DNS resolution for a host name is handled separately from routing.
+Your administrator may have configured a
+link:../../install_config/install/prerequisites.html#prereq-dns[DNS wildcard entry]
+that will resolve to the {product-title} node that is running the
+{product-title} router. If you are using a different host name you may
+need to modify its DNS records independently to resolve to the node that
+is running the router.
+
+Each route consists of a route name (limited to 63 characters), service selector,   
+and (optionally) security configuration.
 
 == Routers
 
-An OpenShift administrator can deploy _routers_ in an OpenShift cluster, which
-enable routes link:../../dev_guide/routes.html#creating-routes[created by
-developers] to be used by external clients. The routing layer in OpenShift is
+An {product-title} administrator can deploy _routers_ to nodes in an
+{product-title} cluster, which enable routes
+link:../../dev_guide/routes.html#creating-routes[created by developers]
+to be used by external clients.
+
+
+The routing layer in {product-title} is
 pluggable, and two available link:#available-router-plug-ins[router plug-ins]
-are provided and supported by default.
+are provided and supported.
 
 ifdef::openshift-enterprise,openshift-origin[]
 [NOTE]
 ====
-See the link:../../install_config/install/deploy_router.html[Installation and
+See the link:../../install_config/install/deploy_router.html#[Installation and
 Configuration] guide for information on deploying a router.
 ====
 endif::[]
 
+A router uses the service selector to find the service and the endpoints backing
+the service. Service-provided load balancing is bypassed and replaced with the
+router's own load balancing. Routers watch the cluster API and automatically
+update their own configuration according to any relevant changes to the IP
+addresses of pods that provide the service. Custom routers can be deployed to
+communicate modifications of API objects to an external routing solution.
+
+In order to reach a router in the first place, requests for host names must
+resolve via DNS to a router or set of routers. The suggested method is to define
+a cloud domain with a wildcard DNS entry pointing to a virtual IP (VIP) backed by
```

Comment:
maybe `pointing to one or more virtual IP (VIP) addresses backed ...`? 

I know its implied but just for some clarity ... 
A single VIP along with multiple routers means that there will be just one active router and the 
other router(s) are hot-standby(s)  - aka passive.
With `> 1` VIPs, more than one router could be in the mix and so one could have an "active-active" config (or even more complex setups ala "active-active-passive").